README.md

```diff
@@ -1,5 +1,9 @@
 # On Premise Habitat Builder Depot
 
+## IMPORTANT NOTICE
+
+Please see the `Migrating Package Artifacts to Minio` section if you are re-installing / upgrading your existing On-Premise Depot. The package artifcats are now stored in a Minio instance, and running a migration script will be required in order to properly transition over.
```

Comment:
It might be worth adding some type of date information here, to the effect that these instructions are only necessary if you installed an on-prem depot before mid June 2018.  I could see these instructions being confusing 6 months down the road to people that setup their on-prem depot after Minio was the default.

Reply:
Excellent point - will update